GameProject/Assets/Turn based assets/Scripts/CubeGrower.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Turn_based_assets.Scripts;
+using UnityEngine;
+
+public class CubeGrower : MonoBehaviour, ISelection
+{
+    public void Select()
+    {
+        this.transform.localScale = this.transform.localScale * 3;
```

Comment:
`3` appears to be an arbitrary number. Consider creating a `const int` with a name that helps clarify what that value represents.